ls1intum/Artemis
# Remove the legacy popup when deleting a submission

### Checklist

### Motivation and Context

The current delete-logic for submissions used an old method which broke URLs and subsequently the breadcrumbs. (The URL changes to `https://artemistest.ase.in.tum.de/course-management/(306/file-upload-exercises/7087/participations/322478/submissions//popup:participation/322478/submission/437569/delete)` for example.)

### Description

We now re-use the `jhiDeleteButton` like everywhere else in the application.

### Steps for Testing

1. Create or find an exercise with participations and submissions
2. Delete a submission
3. Check that the submission is deleted, the breadcrumbs still work and that the URL is valid
